README.md

```diff
@@ -1,15 +1,21 @@
 # OpenMLS [![OpenMLS Chat][chat-image]][chat-link]
 
 [![Tests & Checks](https://github.com/openmls/openmls/actions/workflows/tests.yml/badge.svg)](https://github.com/openmls/openmls/actions/workflows/tests.yml)
-[![ARM64 Build Status](https://cloud.drone.io/api/badges/openmls/openmls/status.svg?ref=refs/heads/main)](https://cloud.drone.io/openmls/openmls)
+[![ARM64 Build Status][drone-image]](https://cloud.drone.io/openmls/openmls)
 [![Deploy Docs](https://github.com/openmls/openmls/workflows/Deploy%20Docs/badge.svg)][docs-main-link]
-[![codecov](https://codecov.io/gh/openmls/openmls/branch/main/graph/badge.svg?token=5SDRDRTZI0)](https://codecov.io/gh/openmls/openmls)
+[![codecov][codecov-image]](https://codecov.io/gh/openmls/openmls)
 [![OpenMLS List][list-image]][list-link]
 [![Docs][docs-main-badge]][docs-main-link]
 ![Rust Version][rustc-image]
 
 A WIP Rust implementation of [Messaging Layer Security](https://github.com/mlswg/mls-protocol/blob/master/draft-ietf-mls-protocol.md) based on draft 12+.
 
+### Documentation
+
+A user manual detailing how basic MLS operations can be performed using OpenMLS
+can be found [here][book-link]. More detailed documentation on OpenMLS' public
+API can be found [here][docs-main-link].
+
```

Comment:
It would be good to mention that there are two version of both sources:
 - One corresponding to the latest release on crates.io (link to docs.rs & openmls.tech/book)
 - One corresponding to the `main` branch (link to openmls.tech/openmls/book & openmls.tech/openmls/docs)